srcpkgs/giara/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'giara'
+pkgname=giara
+version=1.0.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake gettext blueprint-compiler glib-devel"
```

Comment:
cmake is usually just an alternative to pkg-config with the meson build style, is it really required?